e2e-tests/playwright.config.ts

```diff
@@ -48,21 +48,6 @@ const config: PlaywrightTestConfig = {
         ...devices["Desktop Chrome"],
       },
     },
-
-    {
-      name: "webkit",
-      use: {
-        ...devices["Desktop Safari"],
-      },
-    },
-
-    /* Test against branded browsers. */
-    {
-      name: "Microsoft Edge",
-      use: {
-        channel: "msedge",
-      },
-    },
```

Comment:
Curious, why were these removed?

Reply:
So these browsers are not supported while running in linux machine, locally they pass, but in pipeline they fail :(

Reply:
Ah, they are platform dependent as well. I see UI lib tests on Chromium only also, so I think this is fine.